### DISCUSSION TITLE
Antmedia Video gets rotated by 90 degree

### DISCUSSION THREAD

**Discussion by Samriddha01** [ant-media/Ant-Media-Server#5634] [Q&A]
I have used the Antmedia WebRTC-Android-SDK source code and built an android app for my use. I just used it to record a video and later play it and save it.
Now, when i am doing that, all the things recording, publishing etc. are working fine but the video gets rotated by 90 degree while being saved. i tried multiple devices and it is happening everywhere.

**Comment by USAMAWIZARD** [comment]
Can you please try setting this and see if it fixes the issue 
I am referring to [this](https://github.com/ant-media/WebRTC-Android-SDK/pull/16) pr 

`webRTCClient.setRemoveVideoRotationExtention(true);`
